admin-data-explorer-play-server/src/test/scala/net/wiringbits/webapp/utils/admin/modules/DataExplorerTestModule.scala

```diff
@@ -11,6 +11,9 @@ class DataExplorerTestModule extends AbstractModule {
   }
 
   val settings: List[TableSettings] = List(
-    TableSettings("users", "user_id")
+    TableSettings("users", "user_id"), // "UUID" default
+    TableSettings(tableName = "uuid_table", primaryKeyField = "id", primaryKeyDataType = "UUID"), // explicit default
+    TableSettings(tableName = "serial_table", primaryKeyField = "id", primaryKeyDataType = "Int"),
+    TableSettings(tableName = "big_serial_table", primaryKeyField = "id", primaryKeyDataType = "Int"),
```

Comment:
big serial requires a long data type, we are not handling such a case right now, hence, we can remove this table and keep `serial_table`/`uuid_table`

Reply:
I wrote tests for the big_serial_table that all passed, since they're all currently using very small integers, so maybe I should write some specific tests for the BigInt/Long. Per your recommendation, I just added the ADT/enum to enforce correct values, and all tests passed. (Actually, it was the first thing I tried a couple weeks ago, but I had some trouble back then)